### PR TITLE
Fix flake8

### PR DESCRIPTION
**Pull request by GuillaumeSeren**
Hey,
so as discussed in #296 here the individual fix for the flake8 linter.

I kept one error ignored (E501) because there is a lot of them (83).

### CODE REVIEW

**Review by flokli — COMMENTED** [review]
LGTM, apart from a minor nit.

File: afew/tests/test_utils.py [hunk 1]
```diff
@@ -1,9 +1,6 @@
 #
```
[the remark this answers is not in the document]
Comment (GuillaumeSeren) [reply afew/tests/test_utils.py:1]: Ok I add that change to the branch

**Comment by GuillaumeSeren** [comment]
Hey @flokli I have upgraded the branch with your feedback

**Review by flokli — CHANGES_REQUESTED** [review]
Please change the commit message to:

```
afew.tests.test_utils: remove

This removes the test, which was only importing the file, and didn't contain any test cases.
```

**Comment by GuillaumeSeren** [comment]
> Please change the commit message to:

Done !

**Review by flokli — APPROVED** [review]

**Pull request merged**